doc/OnlineDocs/contributed_packages/sensitivity_toolbox.rst

```diff
@@ -21,73 +24,65 @@ We will start with a motivating example:
 
 Here :math:`x_1`, :math:`x_2`, and :math:`x_3` are the decision variables while :math:`p_1` and :math:`p_2` are parameters. At first, let's consider :math:`p_1 = 4.5` and :math:`p_2 = 1.0`. Below is the model implemented in Pyomo.
 
-.. doctest::
-
-    Import Pyomo and sipopt from the sensitivity toolbox
+.. doctest:: python
 
+    # Import Pyomo and sipopt from the sensitivity toolbox
     >>> from pyomo.environ import *
-    >>> from pyomo.contrib.sensitivity_toolbox.sens import sipopt
+    >>> from pyomo.contrib.sensitivity_toolbox.sens import sensitivity_calculation
 
-    Create a concrete model
+    >>> sipopt_available = SolverFactory("ipopt_sens", validate=False).available()
+    >>> k_aug_available = SolverFactory("k_aug", validate=False).available()
+    >>> dot_sens_available = SolverFactory("dot_sens", validate=False).available()
 
+    # Create a concrete model
     >>> m = ConcreteModel()
 
-    Define the variables with bounds and initial values
-
+    # Define the variables with bounds and initial values
     >>> m.x1 = Var(initialize = 0.15, within=NonNegativeReals)
     >>> m.x2 = Var(initialize = 0.15, within=NonNegativeReals)
     >>> m.x3 = Var(initialize = 0.0, within=NonNegativeReals)
 
-    Define the parameters
-
+    # Define the parameters
     >>> m.eta1 = Param(initialize=4.5,mutable=True)
     >>> m.eta2 = Param(initialize=1.0,mutable=True)
 
-    Define the constraints and objective
-
+    # Define the constraints and objective
     >>> m.const1 = Constraint(expr=6*m.x1+3*m.x2+2*m.x3-m.eta1 ==0)
     >>> m.const2 = Constraint(expr=m.eta2*m.x1+m.x2-m.x3-1 ==0)
     >>> m.cost = Objective(expr=m.x1**2+m.x2**2+m.x3**2)
 
 
-The solution of this optimization problem is :math:`x_1^* = 0.5`, :math:`x_2^* = 0.5`, and :math:`x_3^* = 0.0`. But what if we change the parameter values to :math:`\hat{p}_1 = 4.0` and :math:`\hat{p}_2 = 1.0`? Is there a quick way to approximate the new solution :math:`\hat{x}_1^*`, :math:`\hat{x}_2^*`, and :math:`\hat{x}_3^*`? Yes! This is the main functionality of sIPOPT.
+The solution of this optimization problem is :math:`x_1^* = 0.5`, :math:`x_2^* = 0.5`, and :math:`x_3^* = 0.0`. But what if we change the parameter values to :math:`\hat{p}_1 = 4.0` and :math:`\hat{p}_2 = 1.0`? Is there a quick way to approximate the new solution :math:`\hat{x}_1^*`, :math:`\hat{x}_2^*`, and :math:`\hat{x}_3^*`? Yes! This is the main functionality of sIPOPT and k_aug.
 
 Next we define the perturbed parameter values :math:`\hat{p}_1` and :math:`\hat{p}_2`:
 
-.. doctest::
+.. doctest:: python
 
     >>> m.perturbed_eta1 = Param(initialize = 4.0)
     >>> m.perturbed_eta2 = Param(initialize = 1.0)
 
-And finally we call sIPOPT:
+And finally we call sIPOPT or k_aug:
 
-.. doctest::
-    :skipif: not sipopt_available
+.. doctest:: python
+    :skipif: not sipopt_available or not k_aug_available or not dot_sens_available
```

Comment:
Failing with `k_aug_available` not defined. Is something wrong with this syntax, or my assumption that previously declared variables will be in scope?